c2corg_api/scripts/migration/documents/waypoints/huts.py

```diff
@@ -94,6 +95,16 @@ def get_document_locale(self, document_in, version):
             access_period=document_in.staffed_period
         )
 
+    def set_custodianship(self, document_in):
```

Comment:
The goal of the function is to set the value of the ``custodianship`` attribute, isn't it?